cooler/create/_create.py

```diff
@@ -1270,19 +1227,19 @@ def create_scool(cool_uri, bins_dict, cell_name_pixels_dict, columns=None,
             info["metadata"] = metadata
         write_info(h5, info, True)
 
-    # sort bins_dict and cell_name_pixels_dict
-    # to guarantee matching keys
-    bins_dict_key_list = sorted(bins_dict)
-    cell_name_pixels_key_list = sorted(cell_name_pixels_dict)
-    for key_bins, key_pixel in zip(bins_dict_key_list, cell_name_pixels_key_list):
-        if key_bins != key_pixel:
-            raise ValueError('Bins and pixel dict are not in the same order!')
-    # for cell_pixel, cell_name in zip(pixels_list, cell_name_list):
-        if '/' in key_pixel:
-            cell_name = key_pixel.split('/')[-1]
+    # Append single cells
+    for key in cell_names:
+        if '/' in key:
+            # kill any slashes in the name, not just the leading one
+            cell_name = key.replace('/', '')
```

Comment:
Good point. I can revert, but my intent here was to prevent cells from ending up in nested groups if there happens to be a slash somewhere inside the name. Do we want to allow it or catch it and raise an error? Warning?

Reply:
My strategy would prevent this too. It is only accepting the string after the last `/` in the name, isn't?

Reply:
:facepalm: that's right.. ok, i'll revert